vitest.config.ts

```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from 'vite'
+
+export default defineConfig({
+  test: {
+    include: ['./tests/**/*.test.ts'],
+    exclude: ['./old-tests'],
```

Comment:
Same here.

Also, more as an opinion, how would you see to separate the tests in `tests/unit` and `test/api` (or `test/e2e`) folders? I feel it can be more clearly organized like that

Reply:
I wouldn't be against - to me I also prefer that practice as it's more modular, and at the end, a unit test it's tightly couple to the "component" that is testing.

Only thing is that we'd be introducing an inconsistency with the other packages as they have it in a `test` folder - but we can pioneer the new approach from js-client, and later in time apply it to other pkgs when we update tests there. What do you think?

Reply:
Agree. Mmm, since we already introduced the pnpm workspaces setup, I think it's worth start propagating these new practices down.